code/game/objects/objs.dm

```diff
@@ -123,7 +123,7 @@
 			if ((M.client && M.machine == src))
 				is_in_use = TRUE
 				ui_interact(M)
-		if(isAI(usr) || iscyborg(usr) || IsAdminGhost(usr))
+		if(issilicon(usr) || IsAdminGhost(usr))

```

Comment:
Are you sure this is a safe change

Reply:
I made this change under the advisement of @AnturK